Octokit/Models/Request/CollaboratorRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class CollaboratorRequest
```

Comment:
I'm actually wondering whether we should just be using the existing `RepositoryPermissionRequest` request class for this (currently used in `TeamsClient.AddRepository()`).  Not sure what @shiftkey thinks about re-using a request model in different clients (eg TeamsClient and RepoCollaboratorsClient) vs keeping them separate, incase they ever diverge